mempool/src/shared_mempool/network.rs

```diff
@@ -471,6 +471,10 @@ impl<NetworkClient: NetworkClientInterface<MempoolSyncMsg>> MempoolNetworkInterf
                             }
                         })
                         .collect::<Vec<_>>();
+                    info!(
```

Comment:
Is this log message needed? Remove if not.

Reply:
Removed